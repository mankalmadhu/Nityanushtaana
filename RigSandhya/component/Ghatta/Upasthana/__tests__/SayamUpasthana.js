import React from 'react';
import ಸಾಯಮುಪಸ್ಥಾನ from '../SayamUpasthana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');
jest.mock('../../../Vishaya/Viniyoga',()=>'ವಿನಿಯೋಗ');
jest.mock('../JaataVedase',()=>'ಜಾತವೇದಸೇ');
jest.mock('../Tachayo',()=>'ತಚ್ಛ೦ಯೋ');
jest.mock('../NamoBramhane',()=>'ನಮೋಬ್ರಹ್ಮಣೇ');
jest.mock('../SayamDishaNamskara',()=>'ಸಾಯ೦ದಿಶಾನಮಸ್ಕಾರ');
jest.mock('../DevataNamskara',()=>'ದೇವತಾನಮಸ್ಕಾರ');
jest.mock('../Niragamana',()=>'ನಿರ್ಗಮನ');
jest.mock('../Pravara',()=>'ಪ್ರವರ');
jest.mock('../AasatyaLokaat',()=>'ಆಸತ್ಯಲೋಕಾತ್');

test("ಸಾಯಮುಪಸ್ಥಾನ", () => {
   expect(Renderer.create(
    <ಸಾಯಮುಪಸ್ಥಾನ />
  ).toJSON()).toMatchSnapshot();

});