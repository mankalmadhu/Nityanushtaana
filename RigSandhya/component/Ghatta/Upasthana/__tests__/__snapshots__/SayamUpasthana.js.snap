// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸಾಯಮುಪಸ್ಥಾನ 1`] = `
<View>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    ಉಪಸ್ಥಾನ
  </ವಿಷಯ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
  >
    (ಪಶ್ಚಿಮಾಭಿಮುಖವಾಗಿ ನಿ೦ತು)
  </ವಿಷಯ>
  <ಜಾತವೇದಸೇ />
  <ತಚ್ಛ೦ಯೋ />
  <ನಮೋಬ್ರಹ್ಮಣೇ />
  <ವಿನಿಯೋಗ>
    (ಮೂರು ಬಾರಿ)
  </ವಿನಿಯೋಗ>
  <ಸಾಯ೦ದಿಶಾನಮಸ್ಕಾರ />
  <ದೇವತಾನಮಸ್ಕಾರ />
  <ನಿರ್ಗಮನ />
  <ನಮೋಬ್ರಹ್ಮಣೇ />
  <ಪ್ರವರ />
  <ಆಸತ್ಯಲೋಕಾತ್ />
</View>
`;
